src/pages/components/IconButton.qml

```diff
@@ -33,4 +33,11 @@ IconButton {
     }
 
     Behavior on opacity { FadeAnimation {} }
+
+    Rectangle {
+        visible: url.substring(0, 18) == "https://jolla.com/"
```

Comment:
What is this?

Reply:
This is debug code for making design easier (displaying a border around elements to help placement). It will be removed in the final version, and I should have added a comment to that effect. You'll find other Rectangle elements elsewhere for the same purpose, all will be removed.

Reply:
These debug things should all be removed now.